lib/scout_apm/config/coercions.ex

```diff
@@ -24,4 +24,13 @@ defmodule ScoutApm.Config.Coercions do
   def boolean(_), do: :error
 
 
+  def json(json) when is_list(json), do: {:ok, json}
+  def json(json) when is_map(json), do: {:ok, json}
+  def json(json) when is_binary(json) do
+    case Poison.decode(json) do
```

Comment:
I imagine decode hasn't changed in any version, but note that we are intentionally super loose in our Poison requirement:

https://github.com/scoutapp/scout_apm_elixir/blob/master/mix.exs#L36

Can you spot check older / newer versions to check that Poison hasn't changed `decode`?

Reply:
@cschneid looks like we're safe